homeassistant/components/homekit_controller/strings.json

```diff
@@ -14,7 +14,8 @@
         "title": "Pair with a device via HomeKit Accessory Protocol",
         "description": "HomeKit Controller communicates with {name} over the local area network using a secure encrypted connection without a separate HomeKit controller or iCloud. Enter your HomeKit pairing code (in the format XXX-XX-XXX) to use this accessory. This code is usually found on the device itself or in the packaging.",
         "data": {
-          "pairing_code": "Pairing Code"
+          "pairing_code": "Pairing Code",
+          "allow_invalid_setup_codes": "Allow pairing with invalid setup codes."
```

Comment:
We are using both `invalid` and `insecure`. Is it actually invalid, or is it just insecure? If it's just insecure, we should also update the label to be "Allow pairing with insecure setup codes"

Reply:
Insecure makes more sense here.

The logic was these are `trivial`, therefore `insecure`, therefore `invalid`